src/main/java/TaskManager.java

```diff
@@ -0,0 +1,90 @@
+import java.util.ArrayList;
+
+public class TaskManager {
+    private ArrayList<Task> taskList;
+
+    public TaskManager() {
+        this.taskList = new ArrayList<Task>();
+    }
+
+    public void parseCommand(String command) throws DukeException {
+        if (command.startsWith("list")) {
+            printList();
+        } else if (command.startsWith("done ")) {
+            String[] temp = command.split(" ");
+            int doneIndex = Integer.parseInt(temp[1]) - 1;
+            markTaskDone(doneIndex);;
+        } else if (command.startsWith("delete ")) {
+            String[] temp = command.split(" ");
+            int delIndex = Integer.parseInt(temp[1]) - 1;
+            deleteTask(delIndex);;
+        } else if (command.startsWith("deadline")) {
+            if (command.length() <= 9){
```

Comment:
Perhaps command.strip() could be used to remove the empty spaces after "deadline"? Was thinking how the code will work if someone input "deadline    "

Reply:
If someone input "deadline ", which has a length of 9, a `DukeException` is thrown. Nonetheless, this code is now deprecated and I have pushed a new version.